sirepo/simulation_db.py

```diff
@@ -676,14 +676,14 @@ def sim_data_file(sim_type, sim_id):
     return simulation_dir(sim_type, sim_id).join(SIMULATION_DATA_FILE)
 
 
-def simulation_dir(simulation_type, sid=None):
+def simulation_dir(simulation_type, sid=None, uid=None):
```

Comment:
Could we not add a "acting on behalf of user" feature so that we don't have to pass uid everywhere?  I think this would be more secure, and maybe build it into sirepo.auth.

Let's discuss.